runtime/integration-tests/build.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+use std::{env, fs, path::PathBuf, process::Command};
+
+const LP_SOL_SOURCES: &str = "LP_SOL_SOURCES";
+
+fn main() {
+	let paths = fs::read_dir("./submodules/")
+		.expect("Submodules directory must exist for integration-tests");
+	let out_dir = env::var("OUT_DIR").expect("Cargo sets OUT_DIR environment variable. qed.");
+
+	/*
+	match Command::new("git")
+		.args(&["pull", "--all", "--recurse-submodules=yes"])
```

Comment:
Note - Locally, I had to run a `git submodule update --init --recursive` first.

Reply:
Do we want that in or should everybody pull the deps themselves?

Reply:
Agree with @cdamian that we should modify the message.